test/sql/udf_function/test_vectorized_udf_function.cpp

```diff
@@ -0,0 +1,179 @@
+#include "catch.hpp"
+#include "test_helpers.hpp"
+#include "udf_functions_to_test.hpp"
+
+using namespace duckdb;
+using namespace std;
+
+TEST_CASE("Vectorized UDF functions", "[udf_function]") {
```

Comment:
Could you add a test with 4 or more input columns? Does not need to be for every type, one function will suffice.